packages/perseus/src/widgets/group.jsx

```diff
@@ -129,78 +175,13 @@ class Group extends React.Component<Props> {
             </div>
         );
     }
-
-    change: ($FlowFixMe, $FlowFixMe, $FlowFixMe) => $FlowFixMe = (...args) => {
-        return Changeable.change.apply(this, args);
-    };
```

Comment:
I noticed these methods at the top and wondered if they were new. Now I get here and see you just moved them. My kingdom for Github noticing and pointing out code moves (Phabricator did!)

Reply:
Yeah, I try not to make unnecessary changes, but this was so I could get rid of `/* eslint-disable react/sort-comp */`. Figured if we have this rule we might as well abide by it.